src/Emulator/Peripherals/Peripherals/Network/K6xF_Ethernet.cs

```diff
@@ -0,0 +1,839 @@
+                    .WithFlag(1, out etherEnabled, FieldMode.Write | FieldMode.Read, name: "ETHEREN")
+                    .WithFlag(0, FieldMode.Write,
+                        writeCallback: (_, b) =>
+                        {
+                            if (b) Reset();
+                        },name: "RESET")
+                },
+                {(long)Registers.ReceiveControl, new DoubleWordRegister(this)
+                    .WithTaggedFlag("GRS", 31)
+                    .WithTaggedFlag("NLC", 30)
+                    .WithValueField(16, 14, name: "MAX_FL")
+                    .WithTaggedFlag("CFEN", 15)
+                    .WithFlag(14, out forwardCRC, FieldMode.Write | FieldMode.Read, name: "CRCFWD")
+                    .WithTaggedFlag("PAUFWD", 13)
+                    .WithTaggedFlag("PADEN", 12)
+                    .WithReservedBits(10, 2)
+                    .WithTaggedFlag("RMII_10T", 9)
+                    .WithTaggedFlag("RMII_MODE", 8)
+                    .WithReservedBits(7, 1)
+                    .WithReservedBits(6, 1)
+                    .WithTaggedFlag("FCE", 5)
+                    .WithTaggedFlag("BC_REJ", 4)
+                    .WithTaggedFlag("PROM", 3)
+                    .WithTaggedFlag("MII_MODE", 2)
+                    .WithTaggedFlag("DRT", 1)
+                    .WithTaggedFlag("LOOP", 0)
+                },
+                {(long)Registers.TransmitControl, new DoubleWordRegister(this)
+                    .WithReservedBits(11, 21)
+                    .WithReservedBits(10, 1)
+                    .WithFlag(9, out forwardCRCFromApplication, name: "CRCFWD")
+                    .WithTaggedFlag("ADDINS", 8)
+                    .WithValueField(5, 3, name: "ADDSEL")
+                    .WithTaggedFlag("RFC_PAUSE", 4)
+                    .WithTaggedFlag("TFC_PAUSE", 3)
+                    .WithFlag(2, out fullDuplex, FieldMode.Write | FieldMode.Read, name: "FDEN")
+                    .WithReservedBits(1, 1)
+                    .WithTaggedFlag("GTS", 0)
+                },
+                {(long)Registers.PhysicalAddressLower, new DoubleWordRegister(this)
+                    .WithValueField(0, 32, FieldMode.Write | FieldMode.Read, writeCallback: (_, value) =>
+                    {
+                        lowerMAC = value;
+                        UpdateMac();
+                    }, name: "PADDR1")
+                },
+                {(long)Registers.PhysicalAddressUpper, new DoubleWordRegister(this)
+                    .WithValueField(16, 16, FieldMode.Write | FieldMode.Read, writeCallback: (_, value) =>
+                    {
+                        upperMac = value;
+                        UpdateMac();
+                    },name: "PADDR2")
+                    .WithValueField(0, 16, name: "TYPE")
+                },
+                {(long)Registers.DescriptorGroupUpperAddress, new DoubleWordRegister(this)
+                    .WithValueField(0,32, out upperLimit, name:"GADDR1")
```

Comment:
This and `lowerLimt` seem not to be used anywhere. Is it ok to change them into tags?